src/effector/fork/serialize.ts

```diff
@@ -11,6 +11,9 @@ export function serialize(
   scope: Scope,
   config: {ignore?: Array<Store<any>>; onlyChanges?: boolean} = {},
 ) {
+  if (scope.warnSerialize) {
+    console.error("provided scope cannot be serialized in a reliable way because some stores are missing sid. Please, check that babel-plugin is working or provide sid manually");
```

Comment:
Please reduce the length of the text. The message should fit into one neat sentence.
Libraries like react have a huge dev bundle with obscure error messages in production, nevertheless, the effector has only one bundle, so must keep track of the size of it